nsv13/code/game/objects/items/religion.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/banner/munitions
+	name = "people's republic of munitions banner"
+	desc = "the banner of the people's republic of munitions, masters of arms and ammunition"
+	icon = 'nsv13/icons/obj/banners.dmi'
+	icon_state = "banner_munitions"
+	item_state = "banner_munitions"
+	lefthand_file = 'nsv13/icons/mob/inhands/equipment/banners_lefthand.dmi'
+	righthand_file = 'nsv13/icons/mob/inhands/equipment/banners_righthand.dmi'
+	job_loyalties = list("Master At Arms", "Munitions Technician", "Pilot", "Air Traffic Controller", "Bridge Staff")
```

Comment:
Bridge Staff isn't a part of the munitions department.

Reply:
Under jobs.dm the Bridge Staff role is listed under munitions. Removed Bridge Staff from the list.

Reply:
That is because it has to live somewhere in the code.